docs/sources/shared/configuration.md

```diff
@@ -645,6 +645,15 @@ bloom_build:
     # CLI flag: -bloom-build.planner.max-tasks-per-tenant
     [max_queued_tasks_per_tenant: <int> | default = 30000]
 
+    retention:
+      # Enable bloom retention.
+      # CLI flag: -bloom-build.planner.retention.enabled
+      [enabled: <boolean> | default = false]
+
+      # Max lookback days for retention.
+      # CLI flag: -bloom-build.planner.retention.max-lookback-days
+      [max_lookback_days: <int> | default = 365]
```

Comment:
Can we hide this setting from the docs? Usually you don't need to set this, only in very special edge cases.

Reply:
Because it can cause confusion and could be mixed up with retention period of blooms, which it isn't right?